src/pages/Projects/Components/ProjectList/ProjectList.js

```diff
@@ -14,30 +22,54 @@ class ProjectEntryList extends Component {
   }
 
   render () {
+    const addProject = this.addProject.bind(this);
```

Comment:
Como estamos usando `stage-0` de Babel, podemos escribir esto de la siguiente forma:

`::this.addProject`

Por lo tanto, podemos evitar crear esta const y directamente en el JSX escribir:

`onClick={::this.addProject}>`